src/app/FakeLib/RSyncHelper.fs

```diff
@@ -0,0 +1,19 @@
+﻿[<AutoOpen>]
+/// Contains a task to use [rsync](https://rsync.samba.org/) on Linux.
+module Fake.RSync
+
+/// Executes a rsync command.
+/// ## Parameters
+///  - `source` - The source directory
+///  - `destination` - The target directory
+let RSync (source:string) (destination:string) =
```

Comment:
Shouldn't the order of parameters be `destination` `source` for better composability?